libs/auth-api-lib/seeders/data/scope-vehicles.ts

```diff
@@ -0,0 +1,16 @@
+import { createScope } from './helpers'
+export const up = createScope({
+  // Required:
+  name: '@island.is/vehicles',
+  displayName: 'Vehicles Scope',
+  description: 'Look up vehicular information from Samgöngustofa',
+  // Optional:
+  delegation: {
+    custom: true,
+    legalGuardians: false,
+    procuringHolders: false,
+  },
+  accessControlled: false,
+  addToResource: '@island.is',
+  addToClients: ['@island.is/web'],
```

Comment:
Is the app showing vehicles as well? Or is it on the backlog for the app?

Reply:
But we will be quite alright in leaving it out for a while.

Reply:
That was my thinking, but yeah we will just add it on the app client using the IDS Admin when the time comes.